lib/DxcSupport/dxcapi.extval.cpp

```diff
@@ -6,37 +6,192 @@
 
 namespace dxc {
 
+HRESULT STDMETHODCALLTYPE ExternalValidationHelper::Compile(
+    _In_ IDxcBlob *pSource,         // Source text to compile.
+    _In_opt_z_ LPCWSTR pSourceName, // Optional file name for pSource. Used
+    // in errors and include handlers.
+    _In_opt_z_ LPCWSTR pEntryPoint, // Entry point name.
+    _In_z_ LPCWSTR pTargetProfile,  // Shader profile to compile.
+    _In_opt_count_(argCount)
+        LPCWSTR *pArguments, // Array of pointers to arguments.
+    _In_ UINT32 argCount,    // Number of arguments.
+    _In_count_(defineCount) const DxcDefine *pDefines, // Array of defines.
+    _In_ UINT32 defineCount,                           // Number of defines.
+    _In_opt_ IDxcIncludeHandler
+        *pIncludeHandler, // User-provided interface to handle #include
+    // directives (optional).
+    _COM_Outptr_ IDxcOperationResult *
+        *ppResult // Compiler output status, buffer, and errors.
+) {
+  // First, add -Vd to the compilation args
```

Comment:
```
// First, add -Vd to the compilation args
```

It'd be good if this comment explained why it was doing this.  I can figure out what it's doing by reading the code, but I'm none the wiser as to why it is doing it.